go/app/main.go

```diff
@@ -18,38 +18,46 @@ import (
 
 const (
 	ImgDir = "images"
+	dbType = "sqlite3"
+	dbPath = "mercari.sqlite3"
 )
 
 type Response struct {
 	Message string `json:"message"`
 }
 
-type ResponseItems struct {
-	Items []ItemInfo `json:"items"`
-}
-
-type ItemInfo struct {
-	Name          string `json:"name"`
-	Category      string `json:"category"`
-	ImageFilename string `json:"image_filename"`
-}
-
 func root(c echo.Context) error {
 	res := Response{Message: "Hello, world!"}
 	return c.JSON(http.StatusOK, res)
 }
 
 func addItem(c echo.Context) error {
-	// Open and read items.json
-	var jsonItems ResponseItems
-	jsonFile, err := os.Open("items.json")
+	database, err := db.OpenDb(dbType, dbPath)
+	if err != nil {
+		fmt.Printf("error opening database: %v\n", err)
+		return err
+	}
+	defer database.Close()
+	// read the SQL file
+	itemSql, err := os.Open("db/items.db")
+	if err != nil {
+		fmt.Printf("error opening SQL file: %v\n", err)
+		return err
+	}
+	defer itemSql.Close()
+	// read the content
+	sqlBytes, err := ioutil.ReadAll(itemSql)
+	if err != nil {
+		fmt.Printf("error reading SQL file: %v\n", err)
+		return err
+	}
+	_, err = database.Exec(string(sqlBytes))
```

Comment:
Maybe we can execute it single time as application start. In this case it will be a database initialization/reconcilliation step and it can be called right before we startup the server.